frank/io.py

```diff
@@ -44,6 +45,9 @@ def load_uvtable(data_file):
           Weights on the visibilities, of the form
           :math:`1 / \sigma^2`
     """
+
+    logging.info('  Loading UVTable')
+
```

Comment:
I genuinely think this is unnecessary. If you *really* want to log this function, do it at the debug level, not info

Reply:
I don't see this as unnecessary but as a useful status report to the user. I'd prefer to leave at the info level.

Reply:
The reason that I don't think this is useful is that it is a simple function call. It only does one thing - load a UV table - and you are expecting that when you call the function. So it doesn't need reporting every time you call it. 

As an optional thing for debugging I can see that it would be useful. I can also see it as a useful progress report in fit.py - but if that's what its being used for then it should be there not here. I just don't see it as a useful status report otherwise.

Reply:
Moved to fit.py